src/Components/Components/src/Layouts/PageDisplay.cs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Components.Auth;
+using Microsoft.AspNetCore.Components.Internal;
+using Microsoft.AspNetCore.Components.Layouts;
+using Microsoft.AspNetCore.Components.RenderTree;
+
+namespace Microsoft.AspNetCore.Components
+{
+    /// <summary>
+    /// Displays the specified page component, rendering it inside its layout
+    /// and any further nested layouts, plus applying any authorization rules.
+    /// </summary>
+    public class PageDisplay : IComponent
```

Comment:
Renamed from `LayoutDisplay` to `PageDisplay`, because this now deals with all the concerns of displaying a `@page` component that you reached via routing. It doesn't just wrap the result in the chain of layouts; it now also respects any authorization metadata on the page.